helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml

```diff
@@ -19,6 +19,11 @@ spec:
         class: HIGH
         service: api-server
       record: raw_slo_errors
+    - expr: "scalar(vector(0.999))"
+      labels:
+        class: HIGH
+        service: api-server
+      record: slo_target
```

Comment:
This is the SLO we want for a given service. I added this because it is different for managed apps.